src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -647,7 +669,7 @@ ImmutableArray<Span> GetHighlightedSpans()
                     // Since VS item's display text is created as Prefix + DisplayText + Suffix, 
                     // we can calculate the highlighted span by adding an offset that is the length of the Prefix.
                     return patternMatch.Value.MatchedSpans
-                        .SelectAsArray(s => s.MoveTo(roslynItem.DisplayTextPrefix?.Length ?? 0).ToSpan());
+                        .SelectAsArray(s_highlightSpanGetter, roslynItem);
```

Comment:
alternative: just mark the lambda as `static`.

Reply:
so the static lambda will be compiled into same thing?

Reply:
yup.